DigitalLearningSolutions.Data.Tests/Services/ActivityServiceTests.cs

```diff
@@ -170,6 +176,132 @@ public void GetFilteredActivity_requests_activity_with_correct_parameters()
                 .MustHaveHappened(1, Times.Exactly);
         }
 
+        [Test]
+        public void GetFilterOptions_returns_populated_filter_options()
+        {
+            // Given
+            var jobGroups = JobGroupsTestHelper.GetDefaultJobGroupsAlphabetical().ToList();
+            var expectedFilterOptions = new ReportsFilterOptions(
+                jobGroups,
+                new[] { (1, "Category 1"), (2, "Category 2") },
+                new[] { (2, "A Course"), (1, "B Course") }
+            );
+
+            const int centreId = 1;
+            const int categoryId = 1;
+            var categories = new List<Category>
+            {
+                new Category { CourseCategoryID = 1, CategoryName = "Category 1" },
+                new Category { CourseCategoryID = 2, CategoryName = "Category 2" }
+            };
+            var courses = new List<Course>
+            {
+                new Course { CustomisationId = 1, ApplicationName = "B Course" },
+                new Course { CustomisationId = 2, ApplicationName = "A Course" }
+            };
+            A.CallTo(() => jobGroupsDataService.GetJobGroupsAlphabetical()).Returns(jobGroups);
+            A.CallTo(() => courseCategoriesDataService.GetCategoriesForCentreAndCentrallyManagedCourses(centreId))
+                .Returns(categories);
+            A.CallTo(() => courseDataService.GetCentrallyManagedAndCentreCourses(centreId, categoryId))
+                .Returns(courses);

```

Comment:
Maybe we can break this into a set of smaller tests:
1. Returns expected jobGroups
2. Returns expectedCourseCategories
3. ReturnsAlphabeticallyOrderedCourses